index.d.ts

```diff
@@ -126,6 +132,13 @@ declare namespace contextMenu {
 		*/
 		readonly showCopyImageAddress?: boolean;
 
+		/**
```

Comment:
Not the correct order. The order here should follow the order shown in the menu.